builtin/tag.c

```diff
@@ -106,7 +106,22 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
 {
-	return gpg_verify_tag(sha1, name, GPG_VERIFY_VERBOSE);
+	int flags;
+	flags = GPG_VERIFY_VERBOSE;
+
+	if (fmt_pretty) {
+		verify_ref_format(fmt_pretty);
+		struct ref_array_item *ref_item;
+
+		ref_item = new_ref_item(name, sha1, 0);
+		ref_item->kind = FILTER_REFS_TAGS;
+		show_ref_item(ref_item, fmt_pretty, 0);
+		free_ref_item(ref_item);
+
+		flags = GPG_VERIFY_QUIET;
+	}
```

Comment:
I wouldn't touch those names, although (I would like to). I think they are not super accurate anyway, for instance I couldn't tell the difference between gpg_verify_tag and run_gpg_verify just by their names...